openedx/core/djangoapps/credit/tasks.py

```diff
@@ -231,12 +231,13 @@ def _get_proctoring_requirements(course_key):
     requirements = [
         {
             'namespace': 'proctored_exam',
-            'name': 'proctored_exam_id:{id}'.format(id=exam['id']),
```

Comment:
Where does 'exam' come from?  Better to pass the reference into this helper method instead of assuming it's available in the surrounding context?